src/pseudopeople/noise.py

```diff
@@ -48,30 +48,33 @@ def noise_form(
 
     noise_configuration = configuration[form.name]
     for noise_type in NOISE_TYPES:
-        if noise_type.is_implemented:
-            if isinstance(noise_type, RowNoiseType):
-                # Apply row noise
-                form_data = noise_type(form_data, noise_configuration.row_noise, randomness)
+        if isinstance(noise_type, RowNoiseType):
+            # Apply row noise
+            form_data = noise_type(
+                form_data,
+                noise_configuration.row_noise[noise_type.name],
```

Comment:
This will throw an error if the row noise type doesn't exist in the configuration. This can happen if a `schema_entities.Form` doesn't have all the row noise types defined in `NOISE_TYPES`. We don't expect to have any forms that don't implement all row noise types, but we might down the line.

Reply:
This isn't quite right. It still could fail.